lib/src/screens/dashboard/desktop_widgets/desktop_sidebar/side_menu_item.dart

```diff
@@ -4,34 +4,41 @@ class SideMenuItem extends StatelessWidget {
   const SideMenuItem({
     Key? key,
     required this.onTap,
-    required this.iconPath,
-    required this.isSelected,
+    this.imagePath,
+    this.icon,
+    this.isSelected = false,
   }) : super(key: key);
 
   final void Function() onTap;
-  final String iconPath;
+  final String? imagePath;
+  final IconData? icon;
```

Comment:
you can add an assertion (`assert()`) to the constructor to make sure one of them is not null